src/filesystem-cache.ts

```diff
@@ -134,7 +143,29 @@ export class FilesystemCache {
       }
     }
   }
+  invalidateHandler() {
+    return this.handleInvalidateRequest.bind(this);
+  }
 
+  private async handleInvalidateRequest(ctx: Koa.Context, url: string) {
+    let cacheKey = url
+      .replace(/&?refreshCache=(?:true|false)&?/i, '');
+
+    if (cacheKey.charAt(cacheKey.length - 1) === '?') {
+      cacheKey = cacheKey.slice(0, -1);
+    }
+
+    // remove /render/ from key
+    cacheKey = cacheKey.replace(/^\/render\//, '');
```

Comment:
When I call `/invalidate/<url>`, is there still a `/render` in there?
Was this a copy-and-paste thing?